src/overlays/actors/ovl_Demo_Effect/z_demo_effect.h

```diff
@@ -7,6 +7,7 @@ struct DemoEffect;
 
 typedef void (*DemoEffectActionFunc)(struct DemoEffect*, GlobalContext*);
 
+// 0x144 is curve
```

Comment:
Could you clarify this? Do mean "is a `SkelCurve` struct"? for example?

Reply:
I'll remove both, they were mostly just notes I made in the documentation process.